rmc/utils/mpc.py

```diff
@@ -194,20 +199,19 @@ def prepare_pop_path_ht(
     :param bool overwrite_temp: Whether to overwrite intermediate temporary data if it already exists.
         If False, will read existing intermediate temporary data rather than overwriting.
         Default is False.
-    :param bool overwrite_output: Whether to entirely overwrite final output data if it already exists.
-        If False, will read and modify existing output data by adding or modifying columns rather than overwriting entirely.
-        If True, will clear existing output data and write new output data.
-        The output Tables are the OE-annotated context Tables with duplicated or deduplicated sections
-        and the population/pathogenic variant Table.
-        Default is True.
+    :param do_k_fold_training: Whether to generate k-fold Tables with the training transcripts.
```

Comment:
```suggestion
    :param bool do_k_fold_training: Whether to generate k-fold Tables with the training transcripts.
```

Reply:
Yes, this is cited in our [`Cosmetic fixes`](https://github.com/broadinstitute/rmc_production/issues/108) issue:
```
Remove param types in docstrings
```
So I haven't really been bothering to make these consistent right now